tests/blake2b.rs

```diff
@@ -21,16 +21,20 @@ mod test {
         out_160: Option<String>,
     }
 
-    fn variable_blake2b(input: &[u8], key: &str, size: usize) -> Vec<u8> {
-        let mut digest: VarBlake2b = if key.is_empty() {
-            VarBlake2b::new_keyed(&[], size)
+    fn variable_blake2b(input: &[u8], key: &str, size: usize) -> Option<Vec<u8>> {
```

Comment:
Maybe it is better to remove `key` completely from the function as we are no longer handling keyed-blake.